### PR TITLE
Upgraded layout and representation of Ui.

### PR DESCRIPTION
**Pull request by jltham**
Updated Ui

image

### CODE REVIEW

**Comment by Shivang211** [comment]
Why are their two boxes? on the left, there is too much empty space on the left

File: src/main/java/seedu/address/model/expense/Amount.java [hunk 1]
```diff
@@ -29,7 +29,7 @@
     public Amount(String amount) {
         requireNonNull(amount);
         checkArgument(isValidAmount(amount), MESSAGE_CONSTRAINTS);
-        this.amount = amount;
+        this.amount = String.format("%.2f", Double.valueOf(amount));
```
Comment (lamwj98) [inline src/main/java/seedu/address/model/expense/Amount.java:32]: Just want to check whether the regex already factored in this formatting issue?

Comment (lamwj98) [inline src/main/java/seedu/address/model/expense/Amount.java:32]: Just want to check whether the regex already factored in this formatting issue?

File: src/main/java/seedu/address/model/person/PersonAmount.java [hunk 2]
```diff
@@ -26,7 +26,7 @@
     public PersonAmount(String personAmount) {
         requireNonNull(personAmount);
         checkArgument(isValidPersonAmount(personAmount), MESSAGE_CONSTRAINTS);
-        this.personAmount = personAmount;
+        this.personAmount = String.format("%.2f", Double.valueOf(personAmount));
```
Comment (lamwj98) [inline src/main/java/seedu/address/model/person/PersonAmount.java:29]: Same as above.

Comment (lamwj98) [inline src/main/java/seedu/address/model/person/PersonAmount.java:29]: Same as above.

File: src/main/java/seedu/address/ui/HelpWindow.java [hunk 3]
```diff
@@ -29,6 +29,9 @@
     @FXML
     private Label helpMessage;
 
+    // @FXML
```
Comment (lamwj98) [inline src/main/java/seedu/address/ui/HelpWindow.java:32]: May want to delete these commented out code.

Comment (lamwj98) [inline src/main/java/seedu/address/ui/HelpWindow.java:32]: May want to delete these commented out code.

File: src/main/java/seedu/address/ui/HelpWindow.java [hunk 4]
```diff
@@ -37,6 +40,7 @@
     public HelpWindow(Stage root) {
         super(FXML, root);
         helpMessage.setText(HelpCommand.helpCommandString());
+        // helpImage = new Image(new FileInputStream("/images/coin_sprite.png"));
```
Comment (lamwj98) [inline src/main/java/seedu/address/ui/HelpWindow.java:43]: Same as above.

Comment (lamwj98) [inline src/main/java/seedu/address/ui/HelpWindow.java:43]: Same as above.

**Review by lamwj98 — COMMENTED** [review]
Generally LGTM, just minor issues.

**Review by lamwj98 — APPROVED** [review]

**Pull request merged**